e2e/test/scenarios/joins/reproductions/29795-native-query-join-question-misses-metadata.cy.spec.js

```diff
@@ -0,0 +1,48 @@
+import {
+  restore,
+  visualize,
+  popover,
+  openOrdersTable,
+} from "e2e/support/helpers";
+
+describe("issue 29795", () => {
```

Comment:
```suggestion
describe.skip("issue 29795", () => {
```

Reply:
Please commit this with `[ci skip]` so we don't waste another CI cycle for literally nothing.

Reply:
my bad, used it for testing

Reply:
@nemanjaglumac regarding `[ci skip]`, what do you think if we change folder structure for repro and filter that path from running tests instead of using `[ci skip]` in the commit message?

Reply:
`[ci skip]` is an exception to the rule, so no. I don't think we should overhaul our folder structure to accommodate it.